src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -223,18 +224,18 @@ This does not bypass the age restriction. It just accepts it automatically."</st
     <string name="revanced_enable_mini_player_previous_button_summary">Enables previous button in the miniplayer.</string>
     <string name="revanced_enable_swipe_to_dismiss_mini_player_title">Enable swipe to dismiss miniplayer</string>
     <string name="revanced_enable_swipe_to_dismiss_mini_player_summary">Enables swipe down to dismiss miniplayer.</string>
-    <string name="revanced_enable_zen_mode_title">Enable zen mode</string>
-    <string name="revanced_enable_zen_mode_summary">Changes the player background to light grey to reduce eye strain.</string>
-    <string name="revanced_enable_zen_mode_podcast_title">Enable zen mode in podcasts</string>
-    <string name="revanced_enable_zen_mode_podcast_summary">Zen mode is also applied to podcasts.</string>
-    <string name="revanced_hide_audio_video_switch_toggle_title">Hide audio video switch toggle</string>
-    <string name="revanced_hide_audio_video_switch_toggle_summary">Hides the audio video switch toggle in player.</string>
+    <string name="revanced_enable_zen_mode_title">Enable Zen mode</string>
+    <string name="revanced_enable_zen_mode_summary">Changes the player background color to light grey to reduce eye strain.</string>
+    <string name="revanced_enable_zen_mode_podcast_title">Enable Zen mode in podcasts</string>
+    <string name="revanced_enable_zen_mode_podcast_summary">Also enables Zen mode for podcasts.</string>
+    <string name="revanced_hide_audio_video_switch_toggle_title">Hide Audio / Video switch toggle</string>
+    <string name="revanced_hide_audio_video_switch_toggle_summary">Hides the Audio / Video switch toggle in the player.</string>
     <string name="revanced_hide_comment_channel_guidelines_title">Hide channel guidelines</string>
-    <string name="revanced_hide_comment_channel_guidelines_summary">Hides channel guidelines at the top of the comments section.</string>
+    <string name="revanced_hide_comment_channel_guidelines_summary">Hides the channel guidelines at the top of the comments section.</string>
     <string name="revanced_hide_comment_timestamp_and_emoji_buttons_title">Hide timestamp and emoji buttons</string>
-    <string name="revanced_hide_comment_timestamp_and_emoji_buttons_summary">Hides timestamp and emoji buttons when typing comments.</string>
-    <string name="revanced_hide_fullscreen_share_button_title">Hide fullscreen share button</string>
-    <string name="revanced_hide_fullscreen_share_button_summary">Hides the share button in the fullscreen player.</string>
+    <string name="revanced_hide_comment_timestamp_and_emoji_buttons_summary">Hides the timestamp and emoji buttons when typing comments.</string>
+    <string name="revanced_hide_fullscreen_share_button_title">Hide fullscreen Share button</string>
+    <string name="revanced_hide_fullscreen_share_button_summary">Hides the Share button in the fullscreen player.</string>
```

Comment:
Similar to the `Download action button` on lines 42-46, I have no idea if `fullscreen share button` is a proper noun 🤷